miniapps/src/main/res/layout/screen_destination_two.xml

```diff
@@ -5,21 +5,21 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:orientation="vertical"
-    android:padding="@dimen/dimen_32">
+    android:padding="32dp">
 
     <androidx.emoji.widget.EmojiTextView
         android:id="@+id/titleEmoji"
-        style="@style/Challenge.Title"
+        style="@style/GenericShowcase.Title"
```

Comment:
I think we should add a custom title for this element inside the feature module instead of reusing the generic styles